### PR TITLE
Add sample usage of operator list endpoint

### PR DESCRIPTION
**Pull request by ostap0207**
Example how to use our api and do some actions depending on how many operators are available.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -268,6 +268,28 @@ Operator is considered online whenever he/she is logged into the system. Thus op
...
 ```
 
+> + Sample usage
+
+```
+var usingLiveChat = false
+function onOperatorListChange(operators) {
+    if (operators.length == 0) {
+        usingLiveChat = true;
+        alert("Sorry. There are no available operators. Use live chat.");
```
Comment (indrekj) [inline apiary.apib]: Is it really `live` when there are no operators? seems more like `offline`

**Comment by jzd131** [comment]
I will rewrite this as we need a generic version, I don't want us showing clients using livechat and us on our API doc site

**Comment by paniagua** [comment]
@jzd131 make your first pull request :) i can help you if you want

File: apiary.apib (hunk 1 above)
Comment (ostap0207) [inline apiary.apib]: It's just one of those "livechats", which some used to have before us. Not sure how live it is.

**Comment by willmore** [comment]
@ostap0207 what's the status?

**Comment by ostap0207** [comment]
I updated it with Justin's version. But it got forgotten without review.

**Comment by willmore** [comment]
+1

**Comment by paniagua** [comment]
!squash

**Pull request merged**